tree/dataframe/src/RDFDisplay.cxx

```diff
@@ -89,12 +90,23 @@ bool RDisplayElement::IsEmpty() const
 } // namespace Internal
 
 namespace RDF {
-void RDisplay::AddToRow(const std::string &stringEle)
+
+void RDisplay::EnsureCurrentColumnWidth(size_t w)
 {
    // If the current element is wider than the widest element found, update the width
-   if (fWidths[fCurrentColumn] < stringEle.length()) {
-      fWidths[fCurrentColumn] = stringEle.length();
+   if (fWidths[fCurrentColumn] < w) {
+      if (w > std::numeric_limits<unsigned short>::max()) {
+         w = std::numeric_limits<unsigned short>::max();
+      }
```

Comment:
@bluehood Should one put here error message or assert or it is very improbable situation?

Reply:
I think it's fine to leave things like this, these widths are then passed to `std::setw`, and in case they are shorter than the column name (in case someone has a dataframe column with a name more than 65000 characters long...) the printing will just be uglier -- still better than erroring out (and anyway, the printing will already be screwed up unless your screen is more than 65k columns wide)